gerrymandering/gerrymandering.ipynb

```diff
@@ -97,16 +92,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = dd.io.parquet.read_parquet('./data/census.snappy.parq', engine='pyarrow')\n",
-    "df = df.persist()\n",
-    "census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+    "df = dd.read_parquet('./data/census.snappy.parq', engine='fastparquet')\n",
+    "df = df.persist()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we can datashade and render these points, coloring the points by race:"
+    "Now we will use hvplot with `datashade=True` to render these points efficiently. We also set `dynspread=True` which dynamically decides between showing the distribution of points or specific points based the amount of zoom. At the moment if we want to add tiles and are just using `hvplot.points`, we have to set `tiles=` to a value here; we can't do it later. "
```

Comment:
Would `rasterize=True` work here instead of `datashade=True`?
```suggestion
    "Now we will use hvplot with `rasterize=True` to render these points efficiently. We also set `dynspread=True` which dynamically decides between showing the distribution of points or specific points based the amount of zoom. At the moment if we want to add tiles and are just using `hvplot.points`, we have to set `tiles=` to a value here; we can't do it later. "
```